Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/Behavioral.Automation.Template.Bindings.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <Authors>Quantori Inc.</Authors>
+    <Description>Demo project that can be used as example of test configuration.</Description>
+    <RepositoryUrl>https://github.com/quantori/Behavioral.Automation</RepositoryUrl>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
<PackageReference Include="Selenium.Support" Version="4.1.0" />
    <PackageReference Include="Selenium.WebDriver" Version="4.1.0" />
    <PackageReference Include="SpecFlow" Version="3.9.40" />
Can be removed.

Reply:
Done